### PR TITLE
Staircases Segmentation Using DL Issue #217 

### PR DESCRIPTION
**Pull request by kanishkakataria**
# Pull Request for DL-Simplified 💡

## Issue Title : Staircases Segmentation using DL #217 

- **Info about the related issue (Aim of the project)**: To perform segmentation that is to classify the staircase images as ascending and descending using transfer learning models
- **Name:** Kanishka Kataria
- **GitHub ID:**  kanishkakataria
- **Email ID:** kanishka.kataria99@gmail.com
- **Idenitfy yourself:** SSOC 23' Contributor

Closes: #217 

### Describe the add-ons or changes you've made 📃

I have used transfer learning algorithms like Xception,VGG16,InceptionV3,MobilenetV2 for the training which are pretrained using imagenet weights. For each model I have used the baseline model and have added few conv3D,dense , dropout layers accordingly with batch normalization according to the accuracy of the trained model.
## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️
 I have generated confusion matrix,classification report,accuracy curve and loss curve between training and validation respectively. Other than this I have made predictions on the trained model using  2 test sample for each of four algorithms used.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by abhisheks008** [comment]
Images are not visible in the README.md file, while clicking on them, it is showing that `file not found`. Please have a look at that and fix it.

@kanishkakataria

**Comment by kanishkakataria** [comment]
@abhisheks008 I have rectify the changes and now the images are visible. Sorry for the inconvenience
